code/modules/surgery/surgery.dm

```diff
@@ -121,8 +121,12 @@
 			time_adjust = -20
```

Comment:
Carrions are getting a huge nerf by this please massively increase their surgery things